pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -507,7 +507,7 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 					// TODO: restrict this to the appropriate namespace
 					Verbs:         sets.NewString("proxy"),
 					Resources:     sets.NewString("services"),
-					ResourceNames: sets.NewString("heapster"),
+					ResourceNames: sets.NewString("https:heapster:"),
```

Comment:
Ugh. That's what we get for overloading resource names, I guess :-/
The sooner we can get namespaced service proxying, the better.